example/dev-example/src/browser/frontend-module.ts

```diff
@@ -0,0 +1,25 @@
+/********************************************************************************
```

Comment:
Same here.

Reply:
Good catch. Some files were not properly cleaned up. Should be fixed now